src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/GoGoDuck.java

```diff
@@ -409,25 +405,37 @@ public static int execute(List<String> command) throws Exception {
         builder.redirectErrorStream(true);
 
         final Process process = builder.start();
+        StringBuilder bashLog = new StringBuilder();
 
         try (
             InputStream is = process.getInputStream();
             InputStreamReader isr = new InputStreamReader(is);
             BufferedReader br = new BufferedReader(isr)
         ) {
             String line;
+
             while ((line = br.readLine()) != null) {
                 logger.info(line);
+                bashLog.append(line);
             }
 
             process.waitFor();
 
+            if (process.exitValue() != 0) {
```

Comment:
@anguss00  I tried replicating the [error](https://github.com/aodn/aodn-portal/issues/2263) with my changes without any success. I believe the exception handling will take care of it as earlier the errors were getting logged as info and some exceptions were suppressed. Let me know if you have any concern/question.